core/modules/date/date.elements.inc

```diff
@@ -1633,13 +1633,19 @@ function date_combo_validate($element, &$form_state) {
   // An 'End' date without a 'Start' date is a validation error.
   if ($empty && !empty($item[$to_field])) {
     if (!is_array($item[$to_field])) {
-      form_error($element, t("A start date is required if an end date is supplied for field @field #@delta.", array('@delta' => $field['cardinality'] ? intval($delta + 1) : '', '@field' => $instance['label'])));
+      form_error($element, t("A start date is required if an end date is supplied for field %field #@delta.", array(
+        '@delta' => $field['cardinality'] ? intval($delta + 1) : '',
+        '%field' => $instance['label'],
+      )));
```

Comment:
This is the exact same error thrown as the one the follows it in the `else`. Wondering if we can move this outside the conditional 🤔

Reply:
The issue here is that the second one is within a loop, which is there to check each of its multiple values. So, there is no straightforward way to rewrite this - rewriting will probably take as many lines. Open to suggestions, though.